javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/ATMConfig.qll

```diff
@@ -147,7 +147,7 @@ abstract class AtmConfig extends JS::TaintTracking::Configuration {
    * to this ML-boosted configuration, whereas the unboosted base query does not contain this source and sink
    * combination.
    */
-  predicate getAlerts(JS::DataFlow::PathNode source, JS::DataFlow::PathNode sink, float score) {
+  predicate hasAlert(JS::DataFlow::PathNode source, JS::DataFlow::PathNode sink, float score) {
```

Comment:
Minor: I think I preferred your other idea on the lines of `hasBoostedFlowPath`. To me an alert is the output of a CodeQL query (or indeed any code scanning tool), rather than something we know and reason about within a CodeQL query.

Reply:
I don't love `hasBoostedFlowPath`, but OK, I'll do that 👍

Reply:
Done in 4a6de3e444dc793a57e8c43a91c382edd7ab3cd5